backend/routes/StoriesRoutes.js

```diff
@@ -0,0 +1,62 @@
+const express = require('express')
+const simpleGit = require('simple-git')
+const fs = require('fs')
+const router = express.Router()
+const git = simpleGit()
+const middleware = require("../middleware/auth.js")
+
+const User = require('../models/User.js')
+const Stories = require('../models/Stories.js')
+
+// const {} = require()
+
+router.get('/new-story', function (req, res) {
+  res.send('Add a new story!')
+})
+
+router.post('/new-story', middleware.isLoggedIn, function (req, res) {
+  // Save story
+  let file_name = new Date()
+  file_name = file_name.toISOString().slice(0,10)
+  // todo: add functionality for multiple users to add multiple blogs on the same day
+  let owner_fellow = {
+    id: req.user._id
```

Comment:
You can probably you this library to append a unique string:
https://www.npmjs.com/package/uuid

Reply:
I added the changes, thanks!